pyriemann/spatialfilters.py

```diff
@@ -472,11 +472,11 @@ def fit(self, X, y):
         target -= target.mean()
         target /= target.std()
 
-        C = mean_covariance(X, self.metric)
+        C = mean_covariance(X, metric=self.metric)
         Ce = np.zeros_like(X)
         for i in range(Ce.shape[0]):
             Ce[i] = X[i] * target[i]
-        Cz = mean_covariance(Ce, self.metric)
+        Cz = mean_covariance(Ce, metric=self.metric)
```

Comment:
this shows that this is a non-backward compatible change without deprecation. I would suggest to avoid that.

Reply:
You are the all seeing eye.